core/install.js

```diff
@@ -23,19 +24,25 @@ install = function(name) {
 	console.info('Installation complete.');
 };
 
-exports.requireOrInstall = function(name) {
+exports.requireOrInstall = function(req, name) {
 	try {
-		return require(name);
+		return req(name);
 	}
 	catch(e) {
 		if (!e || !e.code || e.code !== 'MODULE_NOT_FOUND') {
 			throw e;
 		}
+		try {
+			fs.statSync(name);
+		}
+		catch(p) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keyword "catch" must be followed by whitespace.](https://www.codacy.com/app/theprojectshop/Kassy/file/2530104133/issues/source?bid=3282987&fileBranchId=3282997#l38)